drivers/can/can_stm32h7.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 Blue Clover
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+#ifndef ZEPHYR_DRIVERS_CAN_STM32H7_H_
+#define ZEPHYR_DRIVERS_CAN_STM32H7_H_
+
+#include "can_mcan.h"
+
+#define DEV_DATA(dev) ((struct can_stm32h7_data *)(dev)->data)
+#define DEV_CFG(dev) ((const struct can_stm32h7_config *)(dev)->config)
+
+struct can_stm32h7_config {
+	struct can_mcan_msg_sram *msg_sram;
+	void (*config_irq)(void);
+	struct can_mcan_config mcan_cfg;
+	const struct pinctrl_dev_config *pcfg;
+};
+
+struct can_stm32h7_data {
+	struct can_mcan_data mcan_data;
+};
```

Comment:
suggestion: this header seems redundant: all can be part of the source file

Reply:
I've removed the header.